pkgs/tools/misc/man-db/default.nix

```diff
@@ -58,6 +59,7 @@ stdenv.mkDerivation rec {
     "--with-systemdtmpfilesdir=${placeholder "out"}/lib/tmpfiles.d"
     "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
     "--with-pager=less"
+    "--with-col=${util-linuxMinimal}/bin/col"
```

Comment:
Could we take it from PATH like less?

Reply:
The code uses a `PROG_COL` preprocessor define to check for the presence of `col` that will either be set to an absolute path from the PATH of the build environment or to the value of `--with-col`, meaning we pretty much **must** set the `--with-col` flag to something.

I reckon it might work if we use `--with-col=col` and then add it to the PATH in makeWrapper. But haven't tried that yet.

Check out the linked issue for more details and links to the source code.

Reply:
> I reckon it might work if we use --with-col=col and then add it to the PATH in makeWrapper. But haven't tried that yet.

Or we don't add it to wrapper and just leave it up to runtime to be there? I mean less somehow makes it work, too.

Reply:
That breaks the test suite. You can fix it by adding `util-linuxMinimal` to the `nativeCheckInputs` but I think that the failing test suite is a strong indicator that we shouldn't make it possible to run `man` when there isn't a `col` available.

Reply:
The fixup commit now:
- uses `--with-col=col`
- adds `util-linuxMinimal` to the `nativeCheckInputs` to make the tests pass
- adds `util-linuxMinimal` to the `makeWrapper` `--prefix PATH` to avoid man errors when there is no `col` in `PATH`

Not having `util-linuxMinimal` in `makeWrapper` makes e.g. `env PATH=/bin man man | cat` fail. This is a real error that might happen if you use `man <something> | <some process>` in a Nix build sandbox, since `col` is not part of `stdenv`.

I honestly don't think that this fixup commit makes sense. Specifying `--with-col=...` in the `configurePhase` essentially makes `col` a hard dependency, that's required for many non-interactive `man` invocations (anything that pipes man stdout output to another process). And if `col` is a hard dependency, we might as well use the absolute path to `col` in the `--with-col=` option instead of adding it to `PATH` through `makeWrapper`....

Reply:
> I mean less somehow makes it work, too.

I also think that `less` is a fundamentally different case from col since it's only required/added to the processing pipeline of `man` for interactive use, whereas `col` impacts non-interactive uses of `man`.

Reply:
FWIW, I like the original approach, before the fixup, better. It is more explicit and less likely to cause surprises.

Reply:
Ah, if it is a hard dep then that doesn't make sense. I thought we could shave that but if we can't that's fine.